architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,194 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization policy determines whether a user is allowed to perform a given
+link:#action[action]. This allows platform administrators to use
+link:#cluster-policy-and-local-policy[cluster policy] to control who has various
+access levels to the overall OpenShift platform itself. It also allows
+developers to use link:#cluster-policy-and-local-policy[local policy] to control
```

Comment:
There's a few more here too... (cluster policy, local policy) Maybe I'm misunderstanding what a policy is?